docs/samples/specification/azure_key_credential/generated/azure/key/credential/sample/_configuration.py

```diff
@@ -18,6 +19,12 @@
     from typing import Any
 
     from azure.core.credentials import AzureKeyCredential
+def _convert_request(request, files=None):
+    data = request.content if not files else None
+    request = HttpRequest(method=request.method, url=request.url, headers=request.headers, data=data)
+    if files:
```

Comment:
since the properties `files` and `data` are legacy properties, decided not to go with them. With our new request's `content` property, we don't distinguish between "files" and "data", which we need to for old requests. So here, if we're using files (which people very rarely do), we do an additional `set_formdata_body` here